libcloud/compute/drivers/gce.py

```diff
@@ -1942,6 +1942,28 @@ def ex_set_node_labels(self, node, labels):
         self.connection.async_request(request, method='POST', data=body)
         return True
 
+    def ex_set_image_labels(self, image, labels):
+        """
+        Set labels for the specified image.
+
+        :keyword  image: The existing target Image for the request.
+        :type     image: ``NodeImage``
+
+        :keyword  labels: Set (or clear with None) labels for this image.
+        :type     labels: ``dict`` or ``None``
+
+        :return: True if successful
+        :rtype:  ``bool``
+        """
+        if not isinstance(image, NodeImage):
+            raise ValueError("Must specify a valid libcloud image object.")
+        image_name = image.name
```

Comment:
nitpick: I don't see a reason to declare this variable which is only used once and is not shorter than the original.